src/tpm2_pytss/ESAPI.py

```diff
@@ -1770,7 +1770,7 @@ def ecdh_zgen(
 
     def ecc_parameters(
         self,
-        curve_id: TPM2_ECC_CURVE,
+        curve_id: TPM2_ECC,
```

Comment:
Why change these?
```python
TPM2_ECC_CURVE = TPM2_ECC
```

Does it not generate the link correctly, should we do something like TPM2_ECC_CURVE extends TPM2_ECC?

Reply:
Yeah, the link doesn't work for some reason, I'm assuming it's because it's not a type but a variable according to the AST.
Do you mean make TPM2_ECC_CURVE a subclass of TPM2_ECC?

Reply:
Yep, and lets do that for all the "assigned types" to avoid this problem.
```python
class TPM2_ECC_CURVE(TPM2_ECC)
    pass
```